backend/src/main/java/com/mori5/itsecurity/domain/Tag.java

```diff
@@ -0,0 +1,39 @@
+package com.mori5.itsecurity.domain;
+
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.time.Instant;
+import java.util.List;
+
+@Builder
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "tag")
+@Entity
+@EntityListeners(AuditingEntityListener.class)
+public class Tag {
+
+    @Id
+    @Column(length = 36)
+    @GeneratedValue(generator = "uuid_generator")
+    @GenericGenerator(name = "uuid_generator", strategy = "org.hibernate.id.UUIDGenerator")
+    private String id;
+
+    @Column(nullable = false)
+    private String title;
+
+    @ManyToMany
+    private List<Document> containedBy;
```

Comment:
ManyToManynél meg kell adni egy pár dolgot, hogy mi alapján történjen a mappelés, meg ilyenek, van itt egy tök jó leírása a dolognak: https://www.baeldung.com/jpa-many-to-many (2-es pont)